lib/Backend/FlowGraph.cpp

```diff
@@ -172,18 +172,21 @@ FlowGraph::Build(void)
 
     bool assignRegionsBeforeGlobopt = this->func->HasTry() && (this->func->DoOptimizeTry() ||
         (this->func->IsSimpleJit() && this->func->hasBailout) ||
-        this->func->IsLoopBodyInTryFinally());
+        (this->func->IsLoopBodyInTryFinally() && this->func->GetJITFunctionBody()->DoJITLoopBody()));
```

Comment:
>DoJITLoopBody [](start = 83, length = 13)

Wouldn't this always return true if `func->IsLoopBodyInTryFinally()` returns true?

Reply:
Right, its redundant